### PR TITLE
port documentation to rtd

### LINKED ISSUE #19: Having images in the repository will blow up the size of the repo (and it's history)
Must upload images elsewhere, or setup a docs repository to store the images.

### PR DESCRIPTION
**Pull request by ijnek**
Porting documentation to [Soccer Interfaces (RTD)](https://soccer-interfaces.readthedocs.io/en/latest/index.html)  since this repository is growing.
Resolves: #19 

Signed-off-by: Kenji Brameld <kenjibrameld@gmail.com>

### CODE REVIEW

File: soccer_vision_2d_msgs/README.md [hunk 1]
```diff
@@ -2,26 +2,6 @@
...
-
-## Examples
-
-![](images/soccer_vision_2d_msgs_visualized.png)
-
-![](images/obstacle_fieldboundary_markings_visualized.png)
-
-![](images/penalty_spots_and_more.png)
\ No newline at end of file
+For documentation, see [Soccer Vision 2D Msgs](https://soccer-interfaces.readthedocs.io/en/latest/soccer_vision_2d_msgs.html).
```
Comment (jaagut) [inline soccer_vision_2d_msgs/README.md:7]: Is it correct, that this link does not work (yet)?

Comment (ijnek) [reply soccer_vision_2d_msgs/README.md:7]: That link should be working already, it's working for me..

File: README.md [hunk 2]
```diff
@@ -25,18 +25,6 @@ git clone https://github.com/ros-sports/soccer_interfaces.git src/soccer_interfa
...
-**soccer_vision_3d_msgs**
-
-Provides 3d messages for visually detected objects on a soccer field, such as balls, field markings and robots
-
-**soccer_vision_attribute_msgs**
-
-Messages for attributes for objects on a soccer field, such as balls, field markings and robots, that could be visually detected.
-Such attributes can be defined separatly in ``soccer_vision_2d_msgs`` and ``soccer_vision_3d_msgs``, but due to large overlap in information, this package aims to abstract out the attributes that aren't specific to 2d/3d vision.
-``soccer_vision_2d_msgs`` and ``soccer_vision_3d_msgs`` depends on this package.
+For documentation, see [Soccer Interfaces](https://soccer-interfaces.readthedocs.io/en/.latest/).
```
Comment (jaagut) [inline README.md]: Sorry, it is this one...
I get this page:
image

Comment (ijnek) [reply README.md]: Thanks for picking that up, there was a typo in the link, I've fixed it up now.

**Review by jaagut — APPROVED** [review]

**Comment by ijnek** [comment]
Thanks for the review @jaagut

**Pull request merged**